x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/expanded_row.tsx

```diff
@@ -56,6 +57,11 @@ export const ExpandedRow: SFC<Props> = ({ item }) => {
       name: 'JSON',
       content: <JobJsonPane json={item.config} />,
     },
+    {
+      id: 'job-messages',
+      name: 'Messages',
```

Comment:
This needs to be translated like `Job details`, it's just that `JSON` above isn't translated.

Reply:
Added in 82cf66dfffa2c1025f87dc465d6dfac82a63a648
Thanks! 😄